components/bill/BillDetails.tsx

```diff
@@ -52,7 +52,11 @@ export const BillDetails = ({ bill }: BillProps) => {
       {isPendingUpgrade && <PendingUpgradeBanner />}
       {!isCurrentCourt(bill.court) && (
         <Banner>
-          this bill is from session {bill.court} - not the current session
+          {t("bill.banner1")}
```

Comment:
nit: Instead of the non-breaking spaces, can this be an interpolated string like, `this bill is from session {{billCourt}} - not the current session` that we call like `{t("whatever.key", { billCourt: bill.court })}`

Reply:
fixed